LightBulb/LightBulb/ViewController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ViewController.swift
+//  LightBulb
+//
+//  Created by Michael Dippery on 6/18/16.
+//  Copyright © 2016 Flatiron School. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    @IBOutlet weak var lightBulb: UIView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // TODO: Change background color to red
```

Comment:
All the TODO's, I think we should add `lightBulb` view... as in..

```swift
// TODO: Change background color of lightBulb view to red
```

That way, they don't jump right to the TODO's and think they should change the background color of the view.